w03-intro-backend-with-express/d4-dusk-mongoose/exercises.md

```diff
@@ -0,0 +1,21 @@
+# Express Mongoose Exercises
+
+### Base Challenges
+
+**Adding Mongoose to toEatly**
+
+1. Clone [this repo](https://github.com/sf-wdi-22-23/toEatly_mongoose/tree/master) as starter code. Run `npm install`.
+1. Install `nodemon` globally on your computer
+1. Install `mongo` - follow [these instructions](http://docs.mongodb.org/manual/tutorial/install-mongodb-on-os-x/)
```

Comment:
Add a step to check if mongo is installed already (should be) before having them install? 

Could move "Run mongo" step up one and have them install only if that didn't work.